src/models/Popup.js

```diff
@@ -417,12 +470,27 @@ export default class Popup extends Base {
       el.classList.add('cc-invisible')
     }
 
+    const { consents } = this;
+
+    const openConsentsElem = document.getElementById(this.options.consentSettingsElemId);
```

Comment:
Please do not use abbreviations.

```suggestion
    const openConsentsElement = document.getElementById(this.options.consentSettingsElementId);
```